types/hellosign-sdk/index.d.ts

```diff
@@ -686,6 +653,33 @@ declare namespace HelloSign {
         update(clientId: string, options: ApiAppRequestOptions): Promise<ApiAppResponse>;
         delete(clientId: string): Promise<BaseResponse>;
     }
+
+    class HelloSign {
+        constructor(options: HelloSignOptions);
+        account: AccountModule;
+        signatureRequest: SignatureRequestModule;
+        embedded: EmbeddedModule;
+        oauth: OAuthModule;
+        team: TeamModule;
+        template: TemplateModule;
+        reports: ReportsModule;
+        unclaimedDraft: UnclaimedDraftModule;
+        apiApp: ApiAppModule;
```

Comment:
the order here is different than in the source code, which makes it difficult to compare and notice that bulkSendJob is missing.